src/formatDate.js

```diff
@@ -8,7 +8,48 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  if (date.includes('-')) {
+    dateArr = date.split('-');
+  } else if (date.includes('/')) {
+    dateArr = date.split('/');
+  } else {
+    dateArr = date.split('.');
+  }
```

Comment:
The way you're splitting the date string based on the presence of certain characters might be problematic. This is because it assumes that the only separators that can be used are '-', '/' and '.'. But what if the separator is something else? A more flexible approach would be to use the separator from the `fromFormat` array.